api/openapi-spec/v3/api__v1_openapi.json

```diff
@@ -1292,6 +1305,26 @@
         ],
         "type": "object"
       },
+      "io.k8s.api.core.v1.ContainerResizePolicy": {
+        "description": "ContainerResizePolicy represents resource resize policy for a single container.",
+        "properties": {
+          "policy": {
+            "default": "",
+            "description": "Resource resize policy applicable to the above mentioned resource. If not specified, it defaults to RestartNotRequired.",
+            "type": "string"
+          },
+          "resourceName": {
+            "default": "",
+            "description": "Name of the resource type to which this resource resize policy applies. Supported values: cpu, memory.",
+            "type": "string"
+          }
+        },
+        "required": [
+          "resourceName",
+          "policy"
```

Comment:
policy is generated as required?

Reply:
We don't generate default (RestartNotRequired) values. i.e: get pod -oyaml will not show resizePolicy if it was not specified during pod create. We enforce it correctly when specified.